README.md

```diff
@@ -1,73 +1,84 @@
 # A Declaration by the Representatives of the UNITED STATES OF AMERICA, in General Congress assembled.
 
-WHEN in the Course of human Events it becomes necessary for one People to dissolve the Political Bands which have connected them with another, and to assume among the Powers of the Earth the separate & equal Station to which the Laws of Nature and of Nature’s God entitle them, a decent Respect to the Opinions of Mankind requires that they should declare the causes which impel them to the Separation.
+When in the Course of human Events it becomes necessary for one People to dissolve the Political Bands which have connected them with another, and to assume among the Powers of the Earth the separate & equal Station to which the Laws of Nature and of Nature’s God entitle them, a decent Respect to the Opinions of Mankind requires that they should declare the causes which impel them to the Separation.
 
-WE hold these Truths to be self-evident: that all Men are created equal; that they are endowed by their creator with inherent and  inalienable rights; that among these are life, liberty, & the pursuit of happiness: that to secure these rights, governments are instituted among men, deriving their just powers from the consent of the governed; that whenever any form of government becomes destructive of these ends, it is the right of the people to alter or abolish it, & to institute new government, laying it’s foundation on such principles, & organizing it’s powers in such form, as to them shall seem most likely to effect their safety & happiness. Prudence indeed will dictate that governments long established should not be changed for light & transient causes; and accordingly all experience hath shown that mankind are more disposed to suffer while evils are sufferable, than to right themselves by abolishing the forms to which they are accustomed. But when a long train of abuses & usurpations begun at a distinguished period and pursuing invariably the same object, evinces a design to reduce them under absolute despotism, it is their right, it is their duty to throw off such government, & to provide new guards for their future security. Such has been the patient sufferance of these colonies; & such is now the necessity which constrains them to expunge their former systems of government. The history of the present king of Great Britain is a history of unremitting injuries & usurpations, among which appears no solitary fact to contradict the uniform tenor of the rest but all have in direct object the establishment of an absolute tyranny over these states. To prove this let facts be submitted to a candid world for the truth of which we pledge a faith yet unsullied by falsehood.
+We hold these truths to be self-evident:
+that all men are created equal,
+that they are endowed by their Creator with certain unalienable Rights,
+that among these are Life, Liberty and the pursuit of Happiness,
```

Comment:
Locke used the phrase "life, liberty and property", from what I remember. Not "pursuit of property".